challenge-9/submissions/Mgeorg1/solution-template.go

```
@@ -0,0 +1,552 @@
+}
+
+// NewInMemoryBookRepository creates a new in-memory book repository
+func NewInMemoryBookRepository() *InMemoryBookRepository {
+	return &InMemoryBookRepository{
+		books: make(map[string]*Book),
+	}
+}
+
+var (
+	InMemoRepoIsNotInited = errors.New("InMemoryRepository: books is not initialized")
+	InMemoRepoNotFound    = errors.New("InMemoryRepository: book is not found")
```

golangci-lint (2.12.2)

[error] ST1012: error var InMemoRepoNotFound should have name of the form ErrFoo

(staticcheck)